### PR TITLE
Last Jest tests

### PR DESCRIPTION
**Pull request by muenzpraeger**

### CODE REVIEW

File: force-app/main/default/lwc/libsD3/__tests__/libsD3.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        // Clear mocks so that every test run has a clean implementation
+        jest.clearAllMocks();
+    });
+
+    it('contains a svg element for D3', () => {
+        // Create initial element
+        const element = createElement('c-libs-d3', {
+            is: LibsD3
+        });
+        document.body.appendChild(element);
```
Comment (muenzpraeger) [inline force-app/main/default/lwc/libsD3/__tests__/libsD3.test.js]: @trevor-bliss @kevinv11n How should we test this? Adding `lwc:dom` to a tag throws `MutationObserver should not be mocked outside of the jest test environment`.

File: force-app/main/default/lwc/apexWireMethodToFunction/__tests__/apexWireMethodToFunction.test.js [hunk 2]
```diff
@@ -34,7 +35,7 @@ describe('c-apex-wire-method-to-function', () => {
             getContactListAdapter.emit(mockGetContactList);
             return Promise.resolve().then(() => {
                 const detailEls = element.shadowRoot.querySelectorAll('p');
-                expect(detailEls.length).toBe(2);
+                expect(detailEls.length).toBe(USER_COUNT_RESULT);
```
Comment (trevor-bliss) [inline force-app/main/default/lwc/apexWireMethodToFunction/__tests__/apexWireMethodToFunction.test.js]: A good example why extracting the expected values from the test data set would make the tests more resilient. Only thing that would need to be changed is the test name, but you can always rename the test to something more generic about it rendering the contact record data.

File: force-app/main/default/lwc/contactList/__tests__/contactList.test.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+        // Prevent data saved on mocks from leaking between tests
+        jest.clearAllMocks();
+    });
+
+    describe('getContactList @wire data', () => {
+        it('with six records', () => {
```
Comment (trevor-bliss) [inline force-app/main/default/lwc/contactList/__tests__/contactList.test.js]: What is the behavior you're verifying?

File: force-app/main/default/lwc/contactList/__tests__/contactList.test.js [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+                const nameEls = element.shadowRoot.querySelectorAll('p');
+                expect(nameEls.length).toBe(USER_COUNT_RESULT);
+                expect(nameEls[0].textContent).toBe(USER_NAME_RESULT);
+
+                const picEl = element.shadowRoot.querySelector('img');
+                expect(picEl.src).toBe(USER_PIC_RESULT);
+            });
+        });
+
+        it('with no record', () => {
```
Comment (trevor-bliss) [inline force-app/main/default/lwc/contactList/__tests__/contactList.test.js]: What is the behavior you're verifying?

File: force-app/main/default/lwc/libsD3/__tests__/libsD3.test.js (hunk 1 above)
Comment (trevor-bliss) [reply force-app/main/default/lwc/libsD3/__tests__/libsD3.test.js]: Does the ChartJs test work? That also has the `lwc:dom` tag.

File: force-app/main/default/lwc/apexWireMethodToFunction/__tests__/apexWireMethodToFunction.test.js (hunk 2 above)
Comment (muenzpraeger) [reply force-app/main/default/lwc/apexWireMethodToFunction/__tests__/apexWireMethodToFunction.test.js]: Good point!

File: force-app/main/default/lwc/contactList/__tests__/contactList.test.js (hunk 4 above)
Comment (muenzpraeger) [reply force-app/main/default/lwc/contactList/__tests__/contactList.test.js]: Updated text.

File: force-app/main/default/lwc/contactList/__tests__/contactList.test.js (hunk 3 above)
Comment (muenzpraeger) [reply force-app/main/default/lwc/contactList/__tests__/contactList.test.js]: Updated text.

**Pull request merged**